src/components/MetricsPane/SelectedSiteMetrics.jsx

```diff
@@ -83,28 +89,100 @@ export const SelectedSiteMetrics = ({
     updateURLParams(queryParams)
   }
 
-  const handleSampleDateChange = ({ target: { value } }) => {
+  const handleSurveyChange = ({ target: { value } }) => {
     updateCurrentSampleEvent(value)
   }
 
+  const surveysAtSimilarSites = getSurverysAtSimilarSites({
+    projectData: displayedProjects,
+    surveyToCompareTo: selectedSampleEvent,
+  })
+
+  const getProjectSiteOptionLabel = (survey) => `${survey.project_name} - ${survey.site_name}`
+
+  // we only show one survey per site in the project-site drop down since the user will be able to select other surveys at each site from the sample date drop down
+  const surveysAtSimilarSitesLimitedToOneSurveyPerSite = surveysAtSimilarSites
+    .sort((a, b) => new Date(b.sample_date).getTime() - new Date(a.sample_date).getTime()) // we first sort the similar sites so that we get the most recent survey at a site showing as selected in the sample date drop down
+    .reduce(
+      (accumulatedSurveys, newSurvey) => {
+        const doesSurveyFromSameSiteAlreadyExist = accumulatedSurveys.some(
```

Comment:
could use `find` instead of `some` since we're just looking for one

Reply:
As much as I go out of my way to avoid `reduce`, unfortunately we are expecting an array of more than one thing here.

Reply:
ahh ok wasn't sure by the name all good then!